q2_types_genomics/reference_db/tests/test_format.py

```diff
@@ -88,3 +89,44 @@ def test_eggnog_dir_fmt(self):
         fmt_obj = EggnogRefDirFmt(dirpath, mode='r')
 
         fmt_obj.validate()
+
+    def test_eggnog_sequence_taxa_dir_fmt(self):
+        dirpath = self.get_data_path('eggnog_seq_tax')
+        fmt_obj = EggnogSequenceTaxaDirFmt(dirpath, mode='r')
+
+        fmt_obj.validate()
+
+    def test_EggnogRefTextFileFmt_valid(self):
+        filepath = self.get_data_path('eggnog_seq_tax/e5.taxid_info.tsv')
+        fmt_obj = EggnogRefTextFileFmt(filepath, mode='r')
+
+        fmt_obj.validate()
+
+    def test_EggnogRefTextFileFmt_too_many_cols(self):
+        filepath = self.get_data_path('eggnog_seq_tax_bad/too_many_cols.tsv')
+        fmt_obj = EggnogRefTextFileFmt(filepath, mode='r')
+
+        with self.assertRaises(ValidationError):
+            fmt_obj.validate()
+
+    def test_EggnogRefTextFileFmt_invalid_rank(self):
+        filepath = self.get_data_path('eggnog_seq_tax_bad/invalid_rank.tsv')
+        fmt_obj = EggnogRefTextFileFmt(filepath, mode='r')
+
+        with self.assertRaises(ValidationError):
+            fmt_obj.validate()
+
+    def test_EggnogRefTextFileFmt_invalid_taxid(self):
+        filepath = self.get_data_path('eggnog_seq_tax_bad/invalid_taxid.tsv')
+        fmt_obj = EggnogRefTextFileFmt(filepath, mode='r')
+
+        with self.assertRaises(ValidationError):
+            fmt_obj.validate()
+
+    def test_EggnogRefTextFileFmt_invalid_taxid_lineage(self):
+        filepath = self.get_data_path(
+            'eggnog_seq_tax_bad/invalid_taxid_lineage.tsv')
+        fmt_obj = EggnogRefTextFileFmt(filepath, mode='r')
+
+        with self.assertRaises(ValidationError):
+            fmt_obj.validate()
```

Comment:
Similarly to what I already mentioned on the other PR: when testing ValidationErrors please always also validate the message itself.